docs/reference/indices/rollover-index.asciidoc

```diff
@@ -4,8 +4,11 @@
 <titleabbrev>Rollover index</titleabbrev>
 ++++
 
-Assigns an <<indices-aliases, index alias>> to a new index
-when the alias's existing index meets a condition you provide.
+Creates a new index for a rollover target when the target's existing index meets a
+condition you provide. A rollover target can be either an <<indices-aliases, index alias>>
+or a <<indices-datastreams, data stream>>. When targeting an alias, the alias is updated to
```

Comment:
`indices-datastreams` isn't a valid anchor in the docs right now. This is causing the docs CI to fail.

> or a <<indices-datastreams, data stream>>. When targeting an alias, the alias is updated to

It looks like we don't have data stream docs at all atm. It might be good to add those before merging this.

Reply:
@jrodewig, I've added those in https://github.com/elastic/elasticsearch/pull/55557 if you would like to see them. I presume they will need to be merged first to create the appropriate data streams anchors.

Reply:
👍 Awesome. And yes--- they'll need to be merged in before you can reference the anchor here. You may have to update this branch after #55557 merges to get the docs CI to pass.